### PR TITLE
Add copy button to code snippets in docs

### PR DESCRIPTION
**Pull request by anjali411**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#63149 Add copy button to code snippets in docs**

Differential Revision: [D30308891](https://our.internmc.facebook.com/intern/diff/D30308891)

### CODE REVIEW

File: docs/cpp/source/conf.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #
-needs_sphinx = '1.6'
+needs_sphinx = '1.8'
```
Comment (albanD) [inline docs/cpp/source/conf.py:28]: We have 3.5.4 in the requirements.txt. Why not use that here?

**Review by albanD — APPROVED** [review]
Sounds good!

**Review by seemethere — APPROVED** [review]

**Comment by anjali411** [comment]
@anjali411 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30308891).

**Review by albanD — APPROVED** [review]
lgtm

**Pull request closed without merging**